tests/viscoplastic_melt_fraction.prm

```diff
@@ -0,0 +1,121 @@
+# tests the melt fraction calculation in the visco plastic material
+
+set Adiabatic surface temperature          = 1623
+
+# The number of space dimensions you want to run this program in.
+set Dimension                              = 2
+
+# The end time of the simulation. Units: years if the 'Use years in output
+# instead of seconds' parameter is set; seconds otherwise.
+set End time                               = 1000
+set Pressure normalization                 = surface
+set Surface pressure                       = 0
+set Resume computation                     = false
+set Start time                             = 0
+set Use years in output instead of seconds = true
+set Output directory                       = output-vp-melt-fraction
+
+
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators = top, bottom
+  set Model name = initial temperature
+
+  subsection Initial temperature
+    set Maximal temperature = 2773
+    set Minimal temperature = 273
+  end
+end
+
+subsection Boundary composition model
+  set List of model names = initial composition
+end
+
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X extent  = 500e3
+    set Y extent  = 500e3
+  end
+end
+
+subsection Gravity model
+  set Model name = vertical
+
+  subsection Vertical
+    # Magnitude of the gravity vector in $m/s^2$. The direction is always
+    # radially outward from the center of the earth.
+    set Magnitude = 1e-4
+  end
+end
+
+subsection Compositional fields
+  set Number of fields = 2
+  set Names of fields = porosity, bound_fluid
```

Comment:
You can probably remove the bound fluid, right? (if not, we should figure out how to fix that)
```suggestion
  set Number of fields = 1
  set Names of fields = porosity
```

Reply:
I can't remove the bound fluid or porosity right now (I shouldn't need either with just the melt fraction and not transport). I think I can go through and remove the asserts for now if it's using the katz2003 model, but asserts for porosity and periodite will need to be added back in later